packages/cli/__tests__/__mocks__/snapshot/components/button.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports.default = Home;
+require("../stylex_bundle.css");
+var stylex = _interopRequireWildcard(require("@stylexjs/stylex"));
+var _otherStyles = _interopRequireDefault(require("./otherStyles"));
+var _npmStyles = _interopRequireDefault(require("./npmStyles"));
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
```

Comment:
I wonder if for the sake of making these snapshots more readable we can avoid the babel module interop code appearing here

Reply:
Is this an option that can be passed into babel? These files are supposed to be the expected output of the CLI, so I think adding a specific option for tests degrades the integrity of the tests a bit.

Reply:
Yes, this is an option that can be passed into Babel. We don't want any transform other than StyleX to run. So we probably just need to set `babelrc: false` to ensure that is the case.